content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md

```diff
@@ -28,8 +23,6 @@ If you want to make local changes to a gist and push them up to the web, you can
 
 To view a gist's full commit history, click the "Revisions" tab at the top of the gist.
 
-![Gist revisions tab](/assets/images/help/gist/gist_revisions_tab.png)
-
 You will see a full commit history for the gist with diffs.
 
 ![Gist revisions page](/assets/images/help/gist/gist_history.png)
```

Comment:
We need to update this alt text.

Also the image looks a bit blurry to me. Is it 144 dpi as specified [here](https://github.com/github/docs-internal/blob/main/contributing/images-and-versioning.md#technical-specs)?